src/factory_sim/objectives/pick_up_tool_from_holder.xml

```diff
@@ -6,6 +6,12 @@
     _description="Pick up the tool from the tool holder"
   >
     <Control ID="Sequence" name="TopLevelSequence">
+      <Action
```

Comment:
This was moved up because otherwise the vacuum wasn't activated until after the motion, which sometimes did not pick up on the collision and the weld didn't activate.